vendor/vendor-android/ddmlib/src/main/kotlin/com/malinskiy/marathon/android/ddmlib/AndroidDeviceTestRunner.kt

```diff
@@ -104,7 +105,12 @@ class AndroidDeviceTestRunner(private val device: DdmlibAndroidDevice) {
         testBatch: TestBatch
     ): RemoteAndroidTestRunner {
 
-        val runner = RemoteAndroidTestRunner(info.instrumentationPackage, info.testRunnerClass, device.ddmsDevice)
+        val runner = RemoteAndroidTestRunner(
+            info.instrumentationPackage,
+            info.testRunnerClass,
+            device.ddmsDevice,
+            StatusReporterMode.PROTO_STD
```

Comment:
StatusReporterMode.TEXT is deprecated from API 26 and used by default in 3 arguments constructor. StatusReporterMode.PROTO_STD should be better as it will use less data because of binary format rather than text.